src/Components/NoMovieFound.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from 'react';
+import Header from "./Header";
+import { useLocation } from "react-router-dom";
+import { connect } from "react-redux";
+import { getMovies } from "../Actions/actions";
+import Main from "./Main";
+import PropTypes from "prop-types";
+
+
+const useQuery = () => new URLSearchParams(useLocation().search)
+
+
+const NoMovieFound = ({ moviesList, toggleMovieModal }) => {
+    const [searchedFilm, setSearchedFilm] = useState('');
+    const query = useQuery();
+    const filmName = query.get("title");
+    const getFilteredFilms = () => {
+        return filmName ? moviesList.filter(movie => movie.title === searchedFilm) : [];
```

Comment:
I'm not sure, but do we really need `searchedFilm` here? Shouldn't it be `filmName`?